instructions.md

```diff
@@ -0,0 +1,41 @@
+# How to Apply For FDC3 Certification for a Desktop Agent
+
+## 1. Run The Conformance Tests Yourself.
+
+See [Readme](README.md) about how to do this.  
+ 
+## 2. Raise An Issue On This project
+
+[Raise An Issue](https://github.com/finos/FDC3-Conformance-Framework/issues) on the FDC3 Conformance Project.  
+
+In all likelihood, you will need to liase with the maintainers of this project in order to get the conformance tests to work with your desktop agent.  
+
+This will also kick off the process of applying for the "FDC3 Certified" mark.
+
+- Provide details in the issue about how you configured and ran the conformance tests in step 1 above.
+- Provide the test results (number of passes / fails, details of failed tests, versions used) in the issue.
+
+## 3.  Replication
+
+Maintainers of the FDC3 Conformance Framework will aim to replicate your test results.  You must work with them to provide the correct version(s) of your software and resolve any issues around licensing that may arise in order  that they can proceed.  
+
+## 4.  Fees
+
+Since certification requires work from the FINOS team a fee is charged for the certification process.   FINOS members receive certification as a benefit and pay no fees for this process.
+
+## 5.  Failed Tests
+
+If your desktop agent fails any of the conformance tests, the maintainers will inform you via the issue you opened in order that you can make fixes to your software and retry the process.
+
+## 6.  Terms
+
+Once your desktop agent has passed the complete suite of tests, you will be required to accept the [Terms and Conditions](terms-conditions/FDC3-Certified-Terms.md) via the completion of the [Participation Form](terms-conditions/participation-form.md).
+
+## 7.  Badge
+
+On completion of the above steps you will be given your certificate via the [Credly](https://cred.ly) platform for use in your own materials / publicity.
+
+## 8.  Re-Certification
+
+Certification only applies to a single version of your software product.   FINOS reserves the right to allow certification to carry over for minor version changes of your desktop agent on request.
```

Comment:
I would be more specific as per board approval, something like:

"If the product demonstrably uses semantic versioning (or comparable versioning schemes which preserve backward compatibility for classes of versions), re-certification a product within a compatible version class against the same version of the standard will be included in the original certification cost".

Reply:
again, let's try and revise together

Reply:
"Reserving the right" allows us to do that, without having to have an argument about major/minor versions... I think the original makes more sense

Reply:
Updated re-certification verbiage.